blog.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" type="text/css" href="styles.css">
+      <title>Blog</title>
+   </head>
+   <body>
+      <header>
+         <div class="header-holder">
+            <section class="left-side-header">
+               <h3 class="hidden-header">Left Side Header</h3>
+               <img src="images/adoptadog-logo.svg" alt="Adopt a Dog Logo">
+               <p>Seattle's premiere dog adoption agency</p>
+            </section>
+            <section class="right-side-header">
+               <h3 class="hidden-header">Right Side Header</h3>
+               <p><img class= "shopping-cart" src="images/cart-icon.svg" alt="shopping cart"> $1234.56  </p>
+               <button> Checkout</button>
+            </section>
+         </div>
+      </header>
+      <nav>
+   <ul class="header-nav-list-holder">
+            <li class="header-nav-list-item "><a class="needs-slash" href="index.html">Home</a></li>
+            <li class="header-nav-list-item"><a href="dogs.html" class="needs-slash">Dogs</a></li>
+            <li class="header-nav-list-item"><a href="blog.html">Blogs</a></li>
+         </ul>
+      </nav>
+      <!--<div id="wrapper">-->
+      <h2 class="page-header">Adoptadog Blog</h2>
+      <div class="grid-item-container">
+         <article class="blog-main-holder">
```

Comment:
I would suggest using flex for your blog-main-holder. So when pushed to mobile screen size, your text wraps underneath your image.